committee-code-of-conduct/election.md

```diff
@@ -25,18 +25,28 @@ This document outlines the process, for Code of Conduct Committee (CoCC) electio
 
 ### Election process
 
-The steering committee will announce a request for nominations some time in July from any member of the following:
-* steering committee
-* members of the code of conduct committee
-* sig chair
-* sig tech lead
+The steering committee will announce a request for nominations some time in July from anyone meeting the [eligibility for candidacy](#eligibility-for-candidacy) guidelines.
 
 The steering committee will accept nominations for at least 1 week. Though the steering
 committee may choose to accept nominations for longer.  After the announced nomination
 period the steering committee will hold a private ballot election using time-limited
 [Condorcet] ranking use the [Schulze method] to break ties. The steering committee
 will publicly announce the results of the election.
 
+#### Nomination process
+
+Nominations can be collected via any means. In the past a Google form has be used to collect nominations. The form had the following questions which may be used as a template for subsequent elections:
+* Email Address (required)
+* Name of Nominee (required)
+* Email address of Nominee (required)
+* Why would you like to nominate this person for the Code of Conduct Committee? (required)
+* Does the nominee have any experience serving on an Ethics Committee or Code of Conduct Committee? (required)
+* How has this nominee demonstrated the [eligibility for candidacy](#eligibility-for-candidacy) guidelines? (optional)
+
+The nomination form must be communicated via the kdev mailing list and may also be publicized via blogs/social for the purposes of community awareness.
+
+Steering must confirm with all nominees (there were not self nominations) that they accept the nomination prior to the voting process.
```

Comment:
```suggestion
Steering must confirm with all nominees (there were not self nominations) that they accept the nomination along with vetting that nominees support the role of the code of conduct committee, i.e. maintaining a safe and inclusive space for everyone to work on Kubernetes, prior to the voting process.
```

@tashimi Would this change address your comment?

Reply:
An alternative is to add a required question to the nomination form.

Reply:
Yup, I think that does, except I would separate it to make the wording a little more clear, i.e.: 
Prior to the voting process, sterring must: 
- Vet that all nominees support the role of the code of conduct committee, i.e. maintaining a self and inclusive space for everyone to work on Kubernetes.
- Confirm with all nominees (that were not self nominations) that they accept the nomination.

Reply:
Thanks! Updated in place. PTAL